egui/src/containers/area.rs

```diff
@@ -172,6 +172,7 @@ pub(crate) struct Prepared {
     pub(crate) movable: bool,
     enabled: bool,
     drag_bounds: Option<Rect>,
+    dry_run: bool,
```

Comment:
```suggestion
    /// Set the first frame of new windows with anchors.
    ///
    /// This is so that we use the first frame to calculate the window size,
    /// and then can correctly position the window the next frame,
    /// without having one frame where the window is positioned in the wrong place.
    temporarily_invisible: bool,
```

Reply:
The change has been applied.